Chinese/ZZB/zzb.md

```diff
@@ -1,3 +1,6 @@
 至尊宝：谁说我斗鸡眼？我只是把视力集中在一点，以改变我以往对事物的看法！
 
-If you speak English, [go here](../../english/marshmallow.md).
\ No newline at end of file
+紫霞仙子继续说：
+
+[你可知月光宝盒？](Chinese/Xinyang/MyLoveStory.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.